babelsberg/tests.js

```diff
@@ -2061,6 +2061,83 @@ TestCase.subclass('users.timfelgentreff.babelsberg.tests.AutomaticSolverSelectio
         this.assert(man.shirt !== man.pants, "shirt and pants must not have the same color");
         this.assert(man.pants === "black" || man.pants === "blue" || man.pants === "white", "pants should be 'black', 'blue' or 'white'");
     },
+	testFilteringByPriority: function () {
```

Comment:
tabs vs. spaces is horribly messed up in these files. I'd rather do it in another PR to keep the diff readable (yes I know about the spaces flag)